Dockerfile

```diff
@@ -214,7 +217,8 @@ WORKDIR /src
 # Better to avoid running semgrep as root
 # See https://stackoverflow.com/questions/49193283/why-it-is-unsafe-to-run-applications-as-root-in-docker-container
 RUN adduser -D -u 1000 -h /home/semgrep semgrep \
-    && chown semgrep /src
+    && chown semgrep /src \
+    && chown semgrep:semgrep /home/semgrep/bin
```

Comment:
why semgrep:semgrep whereas the other chown are using simply semgrep?
Seems better to be consistent no?

Reply:
@aryx i tried only setting the user to `semgrep` but then the group owner remains as `root`, which i thought might potentially be problematic. i don't know why we did `&& chown semgrep /src` to be honest, because the directory is still owned by `root`, just the files that are mounted at runtime are owned by `semgrep`:
```bash
/src $ ls -la
total 20
drwxr-xr-x    6 root     root           192 Sep 15 12:49 .
drwxr-xr-x    1 root     root          4096 Sep 17 19:51 ..
drwxr-xr-x   12 semgrep  semgrep        384 Sep 15 12:49 .git
-rw-r--r--    1 semgrep  semgrep         58 Sep 15 12:49 README.md
-rw-r--r--    1 semgrep  semgrep       3112 Sep 15 12:49 Sina_Taghizadeh_PortScanner.py
-rw-r--r--    1 semgrep  semgrep       7131 Sep 15 12:49 Sina_Taghizadeh_RoutingSim.py
```